src/modules/block/components/blockDetails/blockDetails.css

```diff
@@ -50,5 +50,21 @@ h1 {
   & .content {
     padding-left: 0;
     padding-right: 0;
+    padding-top: 0;
+  }
+}
+
+.feedback {
+  position: relative;
+}
+
+.blockDetailsWrapper {
+  & .errorFeedbackWrapper {
+    display: flex;
+    align-items: center;
+
+    & > div:first-child {
```

Comment:
The use of element name and pseudo selector can easily get affected by the data structure or component changes. Is this not possible using a class name?

Reply:
I had to do this because the element I was targeting is been abstracted by another component and the component exposed a className prop which did not target my element of interest so in-order to prevent my self from creating a new className prop just for the purpose of targeting this html element I had to do this